qemu/tests/secure_img.py

```diff
@@ -0,0 +1,75 @@
+from virttest import error_context
+
+
+def run_cmd_in_guest(vm, cmd, test, timeout=60):
+    """
+    Run command in the guest
+
+    :params vm: vm object
+    :params cmd: a command needs to be ran
+    :params session: the vm's session
+    :params status: cmd status
+    :params output: cmd output
+    """
+    session = vm.wait_for_login(timeout=timeout)
+    status, output = session.cmd_status_output(cmd, timeout=timeout)
+    test.log.info("The command of '%s' output: %s", cmd, output)
+    if not status:
+        return output
+    else:
+        test.fail("cmd runs failed cmd: %s, output: %s", cmd, output)
+    session.close()
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    1. Create the secure boot params inside of the guest
+    2. Download the certification for secure in side of the guest
+    3. Create the boot image file in side of the guest
+    4. Update the zipl config inside of the guest
+    5. Load updated kernel command inside of the guest and reboot
+    6. Check if the guest is secured
+
+    :params secure_params: params for securing the guest
+    :params download_hkd: certification for securing the guest
+    :params boot_img: command for securing the guest
+    :params zipl_config: command for updating the zipl config
+    :params zipl: command for updating the linux entry
```

Comment:
Usually, with `:params` in the docstring you list parameters of the function signature, here `run`. You can remove these. Instead you might want to consider to use self-explanatory names like "create_params_file_cmd" but also, see my comment above.

Reply:
Updated